luarules/gadgets/game_unit_market.lua

```diff
@@ -0,0 +1,394 @@
+if not Spring.GetModOptions().unit_market then
```

Comment:
delete this file. It is no longer in master, and it's nto in team_transfer anyway